src/NUnitEngine/nunit.engine/Internal/ResultHelper.cs

```diff
@@ -138,15 +145,15 @@ public static XmlNode Aggregate(string elementName, string testType, string name
 
             string status = "Inconclusive";
             //double totalDuration = 0.0d;
-            int testcasecount = 0;
-            int total = 0;
-            int passed = 0;
-            int failed = 0;
-            int inconclusive = 0;
-            int skipped = 0;
-            int asserts = 0;
+            var testcasecount = 0;
+            var total = 0;
+            var passed = 0;
+            var failed = 0;
+            var inconclusive = 0;
+            var skipped = 0;
+            var asserts = 0;
```

Comment:
Remember we had comments from several people about not using var for built-in types. I'm in agreement and I'll incorporate that in an edit of the standards shortly.